TravisScripts/install.sh

```diff
@@ -2,13 +2,35 @@
 
 # Downloads
 echo 'Downloading Unity pkg:'
-curl --retry 5 -o Unity.pkg https://download.unity3d.com/download_unity/24bbd83e8b9e/MacEditorInstaller/Unity-2018.1.9f1.pkg
+
+# curl -v http://license.unity3d.com/ > $(pwd)/license.txt
+# cat $(pwd)/license.txt
+
+# curl -v http://activation.unity3d.com/ > $(pwd)/activation.txt
+# cat $(pwd)/activation.txt
+
+# curl -v http://sl-http.unity3d.com/ > $(pwd)/sl-http.txt
+# cat $(pwd)/sl-http.txt
+
+# curl -v http://developer.cloud.unity3d.com/ > $(pwd)/developer.txt
+# cat $(pwd)/developer.txt
+
+# curl -v http://core.cloud.unity3d.com/ > $(pwd)/core.txt
+# cat $(pwd)/core.txt
+
+# curl -v http://accounts.unity3d.com > $(pwd)/accounts.txt
+# cat $(pwd)/accounts.txt
+
+#2019.1.4
+#curl --retry 5 -o Unity.pkg https://download.unity3d.com/download_unity/ffa3a7a2dd7d/MacEditorInstaller/Unity.pkg
+#2018.2.21
+curl --retry 5 -o Unity.pkg https://download.unity3d.com/download_unity/a122f5dc316d/MacEditorInstaller/Unity.pkg
 if [ $? -ne 0 ]; then { echo "Unity Download failed"; exit $?; } fi
 
-echo 'Downloading StandardAssets pkg:'
+#echo 'Downloading StandardAssets pkg:'
```

Comment:
Should we just remove this?

Reply:
you mean `echo 'Downloading StandardAssets pkg`,  In-fact we can remove comment as `StandardAssets pkg` is not required by the current UnityEditors. Will clean-up the script with the other Travis issues.